ci/recipe/meta.yaml

```diff
@@ -24,6 +24,8 @@ requirements:
     - qiime2 {{ release }}.*
     - q2-types {{ release }}.*
     - fasttree
+    - raxml
```

Comment:
Looks like you might need to `merge` or `rebase` your branch using the latest master from https://github.com/qiime2/q2-phylogeny.

Reply:
Right! I'll have to figure this out and get back to you.

Reply:
Okay, I think I've addressed the conflicts. :-)